doc/specs/OOBE-Dialog.md

```diff
@@ -0,0 +1,112 @@
+|3 | On launch, present a Welcome/Landing page in the OOBE Dialog window as described in [Section 3.1.3.](#313-welcomeoverview-page-content)  | P1 |
+|4 | Traverse the various OOBE Dialog pages via an list view options on the left-hand pane. | P0 |
+|5 | Alternatively traverse the various OOBE Dialog pages via an on-screen next/previous buttons on each page. | P2 |
+|6 | At any time, the user should be able to exit the OOBE dialog by closing the window. | P0 |
+|7 | Alternatively, the user should be able to exit the OOBE dialog by pressing the ESC key. | P1 |
+|8 | Automatically save changes to settings as they are made in the OOBE. | P1 |
+|9 | In the PowerToys settings “General” menu, add a button to re-launch the OOBE dialog under the “Check for updates” button. | P0 |
+|10 | The window is resizable and the content is responsive to being resized. | P0 |
+|11 | The OOBE dialog should have a preset minimum window size to ensure all content can be viable with the need to scroll the page. | P1 |
+
+#### 3.1.2. Page Layout
+
+This section describes the content that should be displayed on a given page for each PowerToys utility.
+
+| No. | Requirement | Pri |
+| - | - | - |
+|1 | Each page of the OOBE dialog should focus on the details and settings of a single PowerToy. | P0 |
+|2 | A page should include textual information that gives an overview of its PowerToy's functionality. | P0 |
+|3 | A page should have a dedicated "How to run/launch" section for explaining the shortcut or actions needed to utilize the PowerToy, emphasizing that users are encouraged to "try it out" while in the OOBE. | P0 |
+|4 | A page should include a hero image/GIF that provides a visual representation of its PowerToy's core functionality. | P0 |
+|5 | A page should include a link to that PowerToys overview page on Microsoft Docs. | P1 |
+|6 | Include a &quot;Settings&quot; button that deep links to the PowerToy's settings page. | P1 |
+|7 | Include a &quot;Pro Tips&quot; section on the page that highlights a useful, yet often undiscovered feature for its PowerToy. | P3 |
+
+#### 3.1.3. Welcome/Overview Page Content
+
+This section describes the content that should be displayed on the Welcome/Overview Page of the OOBE window. [Figure 5.1.2](#512-oobe-welcomeoverview-page) provides a visual of the layout.
+
+| No. | Requirement | Pri |
+| - | - | - |
+|1 | A high level, textual overview of the PowerToys application. | P0 |
+|2 | Link to PowerToys General Settings. | P0 |
+|3 | Link to PowerToys overview on Microsoft Docs. | P0 |
+|4 | Link to PowerToys releases page to showcase latest update changes. | P1 |
+
+## 4. Measure Requirements
+
+| No. | Requirement | Implication | Pri |
+| --- | --- | --- | --- |
+|1 | Date/Time of first-run | Helps to categorize usage and retention trends across various groups of new users. | P0 |
```

Comment:
We already have the telemetry with the information on when PT is first installed (and executed). Not sure how this duplicate metric will help? Can you elaborate?

Reply:
If that's already tracked then there shouldn't be any work required for this. I included it for completeness regarding the various metrics we'd like to access from this OOBE effort.